### PR TITLE
Add pulumi service product page

### PR DESCRIPTION
**Pull request by zchase**
Fixes https://github.com/pulumi/pulumi-hugo/issues/907#issuecomment-999212690

creates page: `/product/pulumi-service`

### CODE REVIEW

**Comment by susanev** [comment]
@gthuang im not really seeing anywhere on the home page, products page, or pricing page where we mention the service by name so we may want to come back and add that in, in the new year so we have places to link to this page.

@zchase i did find a mention of Pulumi Service in parens on the product page could you add a link there, and the Pulumi Console on the pricing page, can we change that to Pulumi Service and link to this new page?

**Comment by susanev** [comment]
@zchase these buttons look pretty broken on a slightly smaller window, can we fix it?
broken-buttons

**Comment by susanev** [comment]
and @zchase on this page only can we increase the customer logo sizes a bit?
customer-logos

**Review by susanev — APPROVED** [review]
thank you for taking this on so quickly; id still like George to review the content ill ping him on slack.

**Comment by gthuang** [comment]
looks good so far!
- I like the screenshot up top, but I'm wondering if it's too big and takes too long to scroll through? Thoughts?
- I feel like it takes too long to get to information about the Service as you scroll down the page. I am wondering if we should swap the positions of the 6 case studies and the section about the open source. If we did that, I'd consider deleting the logos at the bottom of the page.
- Would the text in the expandable sections look better with bullets?
- The guardrails and resource graph screenshots look fuzzy on my screen
- For the "policy pack" screenshot, I think we should rename the "pulumi-internal-policies" to something like "S3 bucket security" so it's more generalizable to customers
- for the CTA, should we also add a "Contact Us" button as secondary?

**Comment by susanev** [comment]
@gthuang are you on safari? i think the svgs don't display well in safari i think we talked about this on the last project we worked on.

**Comment by susanev** [comment]
i do not think we should move the diagram to the top, as we got feedback from folks that it was the least important. if we want to have the open-source section at the top i think we should go back to the older layout that has the small diagram next to the open-source section.

**Comment by susanev** [comment]
@gthuang im fine with the top image height, but i can crop it if you'd like. what part of the image do you want me to remove?

i will try and come back and fix the policy name in the SVG tomorrow, but would rather not block on that.

**Comment by gthuang** [comment]
yeah, I am on safari so that must be why.

what do you think about moving the 6 case studies down then?

**Comment by susanev** [comment]
im good with moving the case studies to wherever you think is best!

**Comment by gthuang** [comment]
let's move the 6 case studies right after the "easiest way to use Pulumi Open source" section. add a header that says "Customers innovating with Pulumi Service". let's also drop the logo section that's at the bottom of the page.

**Comment by susanev** [comment]
okay, heres a summary of the remaining changes to be made

- [ ] link to this new page from the mention of pulumi service in parens on the product page
- [ ] link to this new page from the pricing page where we say Pulumi Console, and also change that to say Pulumi Service
- [ ] fix broken buttons on slightly smaller width screens
- [ ] create a new section directly below the open-source section give it a heading of `Customers innovating with Pulumi Service` and move the case studies there
- [ ] @susanev will fix and push a commit to fix the policies svg

**Comment by gthuang** [comment]
@zchase can you please add also add this FAQ to the pricing page under "Product"? please include a link to the Pulumi Service page in here.

Q: What are Pulumi open source and Pulumi Service?
A: Pulumi's modern Infrastructure as Code CLI and SDK are an open-source project that is supported by an active community. Pulumi Service is a managed service for the open source CLI and SDK. It tracks your infrastructure’s state and coordinates updates with the CLI, which creates or updates resources to reach your infrastructure’s desired state. It also manages secrets, sets up SAML SSO, integrates with CI/CD pipelines, enforces compliance rules, and much more.

You're not required to use Pulumi Service. You can use any cloud or on-premises storage to build and run your own backend.

**Comment by susanev** [comment]
@gthuang could we say supports SAML SSO or similar instead? the service doesn't really set it up for them, they still need to do that part.

**Comment by gthuang** [comment]
yep, just updated the text

File: themes/default/layouts/page/pricing.html [hunk 1]
```diff
@@ -8,13 +8,13 @@ <h2 class="mt-14 text-center">
             Fully managed cloud engineering platform
         </h2>
         <p class="max-w-3xl mx-auto mt-6">
-            The Pulumi Cloud Engineering Platform helps teams adopt Pulumi’s
-            Open Source Infrastructure as Code SDK, and provides easy, robust,
-            and secure state management and workflows.
+            The Pulumi Service is a fully-managed service for the
```
[the remark this answers is not in the document]
Comment (zchase) [reply themes/default/layouts/page/pricing.html:11]: 👍

File: themes/default/layouts/page/pricing.html [hunk 2]
```diff
@@ -8,13 +8,13 @@ <h2 class="mt-14 text-center">
...
+            open source CLI and SDK. It provides secure state & secrets management,
+            collaboration features, CI/CD integrations, compliance enforcement, identity and access controls, and more.
         </p>
         <p class="mt-6">
-            Prefer to self-manage without our platform features? Try
-            <a class="text-blue-600 underline" href="{{ relref . "/docs/get-started" }}">open source</a>.
+            Prefer to self-manage without our platform features? Use Pulumi open source with 
+            <a class="text-blue-600 underline" href="{{ relref . "/docs/intro/concepts/state/#logging-into-a-self-managed-backend" }}">your own backend</a>.
```
[the remark this answers is not in the document]
Comment (zchase) [reply themes/default/layouts/page/pricing.html:17]: 👍

File: themes/default/layouts/page/pricing.html [hunk 3]
```diff
@@ -322,6 +322,25 @@ <h4><a href="#faq-billing" class="text-gray-500">Billing and Support</a></h4>
...
+                         Pulumi's modern Infrastructure as Code CLI and SDK are an open-source project that
+                         is supported by an active community. Pulumi Service is a managed service for the
+                         open source CLI and SDK. It tracks your infrastructure’s state and coordinates
+                         updates with the CLI, which creates or updates resources to reach your infrastructure’s
+                         desired state. It also manages secrets, supports SAML SSO, integrates with CI/CD pipelines,
+                         enforces compliance rules, and much more.
+                    </p>
+                    <p>
+                        You're not required to use <a class="underline" href="{{ relref . "/product/pulumi-service" }}">Pulumi Service</a>.
+                        You can use any cloud or on-premises storage to build and run your own backend.
```
Comment (gthuang) [inline themes/default/layouts/page/pricing.html:339]: @zchase could you please link "run your own backend" to this?
https://www.pulumi.com/docs/intro/concepts/state/

**Pull request merged**